.github/workflows/hil_test_esp-idf.yml

```diff
@@ -40,6 +40,13 @@ jobs:
     needs: build
     runs-on: has_${{ inputs.hil_board }}
 
+    container:
+      image: golioth/golioth-hil-base:latest
```

Comment:
Let's point to a specific version here rather than the latest. That ensures we can keep building on this commit if we make a breaking change to the image later.